### ISSUE TITLE
CSS selector cannot handle plain (unescaped) ampersands in attributes

### ISSUE THREAD

**Issue by DrHaribo** [gatling/gatling#2843]
I'm not sure if it is "correct" to have unescaped ampersands in HTML attributes. But some websites have them and browsers handle it. Some websites even have both ```&``` and ```&amp;``` in the same attribute. But as of version 2.1.7 Gatling's CSS selector cannot handle plain (unescaped) ampersands.

Example:

Say you first receive a webpage containing ```<a href="go?to=foobar&index=null" title="Choice 1">```

Now if I try to follow this link with

```scala
exec(
  http("page with link")
    .get("/page1")
    .check(css("""a[title='Choice 1']""", "href").saveAs("url"))
)
.exec(
    http("follow link")
    .get("${url}")
)
```

then the URL being used for the second request is ```go?to=foobar%E2%88%88dex=null```

See also http://stackoverflow.com/questions/33402172/how-to-extract-href-with-ampersand-using-gatling/33412497

**Comment by slandelle** [comment]
It's a Jodd issue: https://github.com/oblac/jodd/issues/250

**Comment by slandelle** [comment]
Jodd fix is shipped, waiting for Jodd 3.6.7 to be released.
